aria_plugin/environment.py

```diff
@@ -35,7 +36,8 @@ class Environment(object):
 
     def __init__(self):
 
-        self._mk_working_dir()
+        self._workdir = None
```

Comment:
`mk_working_dir` should return the working dir path, and it should be set in the init